src/components/breadcrumb-bar/index.tsx

```diff
@@ -81,9 +83,11 @@ function BreadcrumbBar({
 							weight="medium"
 						>
 							<Elem
-								className={s.breadcrumbText}
-								href={url}
-								aria-current={isCurrentPage ? 'page' : undefined}
```

Comment:
🤔 I still think we should retain the `aria-current` functionality here even though its not a link.